### PR TITLE
Add basic bindings for UBO use.

### PR DESCRIPTION
**Pull request by glennw**
[Review on Reviewable](https://reviewable.io/reviews/servo/gleam/64)

### CODE REVIEW

File: src/gl.rs [hunk 1]
```diff
@@ -248,6 +248,30 @@ pub fn bind_attrib_location(program: GLuint, index: GLuint, name: &str) {
     }
 }
 
+#[cfg(not(target_os="android"))]
+#[inline]
+pub fn get_uniform_block_index(program: GLuint, name: &str) -> GLuint {
+    unsafe {
+        ffi::GetUniformBlockIndex(program, CString::new(name).unwrap().as_ptr())
```
Comment (jdm) [inline src/gl.rs]: This doesn't look like a safe usage of CString. The actual value being pointed to will go out of scope before GetUniformBlockIndex is invoked.

Comment (glennw) [inline src/gl.rs]: Hmm, I did wonder that when I copy/pasted it from other bindings in this file... I'll fix them all up.

Comment (Ms2ger) [inline src/gl.rs]: I don't think it's actually wrong, just dangerous.

**Comment by jdm** [comment]
@pcwalton There was a discussion about how this wasn't really correct...

**Comment by pcwalton** [comment]
@jdm Oops, yes. Submitted a PR to fix these and all the others.

**Pull request merged**